adapters/ix/ix.go

```diff
@@ -100,11 +99,6 @@ func (a *IxAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.P
 		}
 
 		for sizeIndex, format := range unit.Sizes {
-			// ensure request are sent only for valid size as specified in the params
-			if ok := isValidIXSize(format, params.Size); !ok {
```

Comment:
If this is the only place `isValidIXSize` method was used then I'd suggest we also remove that function definition as part of this PR

Reply:
I can't see it anywhere else.

Reply:
Cool. Thanks for conforming @hhhjort. @ix-prebid-support, PR looks good. If you can just please remove the `isValidIXSize` function definition, we can go ahead and merge it. Thanks!

Reply:
Thanks for taking the time to review this PR. We will update the branch so this can be merged.

Reply:
@ix-prebid-support can I ask what's the approx ETA on updating the branch? We're planning on doing a release today and it would be good to have this in as part of that